src/content/app/genome-browser/state/track-panel/trackPanelSlice.ts

```diff
@@ -284,3 +341,44 @@ export const updateCollapsedTrackIds =
       })
     );
   };
+
+const trackPanelSlice = createSlice({
+  name: 'genome-browser-track-panel',
+  initialState: getTrackPanelState() as TrackPanelState,
```

Comment:
1) We shouldn't read browser storage when we are initialising a slice; this code can run on the server-side.

Reply:
I've updated it